src/sdk/pynni/nni/nas/pytorch/callbacks.py

```diff
@@ -51,4 +54,23 @@ def __init__(self, checkpoint_dir, every="epoch"):
         os.makedirs(self.checkpoint_dir, exist_ok=True)
 
     def on_epoch_end(self, epoch):
-        self.trainer.export(os.path.join(self.checkpoint_dir, "epoch_{}.json".format(epoch)))
+        dest_path = os.path.join(self.checkpoint_dir, "epoch_{}.json".format(epoch))
+        _logger.info("Saving architecture to %s", dest_path)
+        self.trainer.export(dest_path)
+
+
+class ModelCheckpoint(Callback):
```

Comment:
can checkpointed model be used for resume? e.g., learning rate, epoch number

Reply:
This part is in another PR. And no, not supported.